site/docs/master/customize-installation.md

```diff
@@ -42,6 +43,16 @@ By default, `velero install` does not install Velero's [restic integration][3].
 
 If you've already run `velero install` without the `--use-restic` flag, you can run the same command again, including the `--use-restic` flag, to add the restic integration to your existing install.
 
+## Enable features
+
+New features in Velero will be released as beta features behind feature flags which are not enabled by default. These features can be enabled using the `--features` flag to the `velero install` command. This flag takes as value, a comma separated list of feature flags to enable. As an e.g. [CSI snapshotting of PVCs][10] can be enabled using `--features=EnableCSI` flags to the `velero install` command.
+
+Similarly, features may be disabled by removing the corresponding feature flags from the `--features` flag.
+
+The feature flags, passed to `velero install` will be passed to the Velero deployment and the `restic` daemon set, if `--use-restic` flag is used.
+
+Enabling and disabling feature flags will require modifying the Velero deployment and also the restic daemonset. This may be done easily by uninstalling and re-installing Velero.
```

Comment:
```suggestion
Enabling and disabling feature flags will require modifying the Velero deployment and also the restic daemonset. This may be done by uninstalling and re-installing Velero, or by editing the resource files in the cluster.
```

Reply:
This could be moved to LN50.

Reply:
PTAL at the new edits 😄